samples/highcharts/demo/polar-advanced/demo.html

```diff
@@ -0,0 +1,94 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/coloraxis.js"></script>
+<script src="https://code.highcharts.com/modules/pattern-fill.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
```

Comment:
All of our demos have first imports of `highcharts` and `highcharts-more` files, then additional modules and like Kacper said we should use also exporting and accessibility modules (please check also if exporting works fully without any bugs).